package.json

```diff
@@ -64,7 +64,8 @@
     "semantic-release-slack-bot": "1.6.2",
     "sequelize": "5.21.3",
     "sequelize-fixtures": "1.1.1",
-    "simple-git": "1.65.0"
+    "simple-git": "1.65.0",
+    "tedious": "11.0.8"
```

Comment:
adding for tests

Reply:
I don't think we are supposed to test things for `sqlite3`/`mariadb` since we do not support it.
I guess you added those for the tests cases, but i'm wondering if that would be possible to remove them.